src/shim.js

```diff
@@ -2,6 +2,36 @@ import webpage from "webpage";
 import system from "system";
 
 /**
+ * Ensure that Function has bind() method (PhantomJS version <= 1.9 support)
+ * This is a Polyfill replacement from MDN.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill}
+ */
+/* eslint-disable no-extend-native, consistent-this, require-jsdoc, no-empty-function, no-invalid-this */
```

Comment:
It's unfortunate that you have to disable all these. But I looked at most of the rules and looks like there isn't any other way around it. :(